### ISSUE TITLE
Using IONIC : Paypal production on IOS not working and getting paypal server error. Working fine with sandbox for ios and android

### ISSUE THREAD

**Issue by mastersoftwaresolutions** [paypal/PayPal-Cordova-Plugin#111]
I am facing strange issue. Sandbox is working fine for bith ios and android versions. When i switched to paypal production, then it stopped working in IOS. But android is working fine and able to make live transactions. Find below screen shot of error i am getting >>
![screenshot 2015-10-29 16 30 06](https://cloud.githubusercontent.com/assets/6111511/10816659/67abf884-7e5a-11e5-9bef-4d29cf2ad1d8.png)

Sandbox :

Android and ios working perfectly fine.

Production:
Android working but ios throwing above error.

Please guide!!

**Comment by ppmtscory** [comment]
@mastersoftwaresolutions I'm wondering if you are running into the same thing as [this issue](https://github.com/paypal/PayPal-iOS-SDK/issues/353).  Do you have a debug ID that I can use to confirm/deny?

**Comment by mastersoftwaresolutions** [comment]
I am sorry but how can i find debug id?

**Comment by ppmtscory** [comment]
There should be a debug ID in the console logs along with the error response.

**Comment by mastersoftwaresolutions** [comment]
I am using Paypal sdk plugin for ionic cordova framework. Let me check if i can any logs there.

**Comment by puneet1106** [comment]
Hi I am also facing the same issue.It is working only once in the noon and after that it is showing me the same alert "PAYER_ACTION_REQUIRED". I am working in sandbox mode. It is working fine in android but giving issue in iOS. My debug id is : 148dc3cdd5954

**Comment by ppmtscory** [comment]
@puneet1106 please see my last response prior to yours on [this issue](https://github.com/paypal/PayPal-iOS-SDK/issues/353).

**Comment by puneet1106** [comment]
@ppmtscory Do i have to use another sdk for paypal?

**Comment by ppmtscory** [comment]
@puneet1106 no - once the changes are pushed, you shouldn't be receiving the same error.

**Comment by puneet1106** [comment]
@ppmtscory So when the changes are going to pushed?Can you give me an idea please?

**Comment by ppmtscory** [comment]
I don't have an ETA at this time.

**Comment by puneet1106** [comment]
Actually i have to submit ipa on iTunes.So that's why i am asking this question so that it might be helpful for me. Do i need to implement something new code my side or will it be from Paypal side?

**Comment by ppmtscory** [comment]
You shouldn't have to change anything in your code as it's a change on our side for one of our internal services.

**Comment by mastersoftwaresolutions** [comment]
Check out this issue :
https://github.com/paypal/PayPal-Cordova-Plugin/issues/106
It says using older version sdk (3.1.12) solves problem for now. i am going to try that now. here is the link to that release version :
https://github.com/paypal/PayPal-Cordova-Plugin/releases/tag/v3.1.12
Let me know if that works for you

**Comment by ppmtscory** [comment]
@mastersoftwaresolutions is yours the same issue whereas it only stopped working after iOS9 but was working prior?

**Comment by mastersoftwaresolutions** [comment]
I have not tried on older version because i have just ios9 at the moment on all devices and machines :(. IOS9 sucks. I am facing many issues just because of this IOS update.

**Comment by ppmtscory** [comment]
Ah, ok.  Let me know if 3.1.12 works for you so that we can conclude that it's the same issue.

**Comment by puneet1106** [comment]
@ppmtscory Yesterday getting issue "PAYER_ACTION_REQUIRED" and today it is like "PAYMENT_CREATION_ERROR - There was a problem setting up this payment. Please visit the PayPal website to check your account. (422) "

Please check this and let me know, what is the issue. After every 3-4 hours i am getting different error.

**Comment by bluk** [comment]
@puneet1106 Regarding the PAYMENT_CREATION_ERROR, it is a new issue as of last night. Please refer to paypal/PayPal-iOS-SDK#355 for further updates for that error.

**Comment by tomwhipple** [comment]
The PAYER_ACTION_REQUIRED error is being tracked as https://github.com/paypal/PayPal-iOS-SDK/issues/353.

Other errors last night or with sandbox today should be resolved, but feel free to re-open/open new issues if not.

**Comment by mastersoftwaresolutions** [comment]
@tomwhipple This issue is still open. Original issue is that sandbox is working but on live ist not working but for IOS only. For android its working fine. So please reopen this issue.

**Comment by mastersoftwaresolutions** [comment]
Guys, it worked for me when i I tried to build my project on different mac with version 10.10.4 (OX X YOSEMITE) with xcode version - 7.0.1(7A1001).

but still not working on my existing machine with MAC version -10.11.1 (OSX EL CAPTAIN) with xcode version- 7.0.1(7A1001).

Anyhow i am not sure how above made difference but it worked. May help you guys as well to solve actual issue.

Thank you
